docs/manuals/source/IntroductionAndTutorial/WhatIsBareos.rst

```diff
@@ -158,6 +161,71 @@ Not all packages are required to run Bareos.
 
 -  On a Backup Administration system you need to install at least **bareos-bconsole** to have an interactive console to the |dir|.
 
+Comparison between Community and Subscription Repositories
+----------------------------------------------------------
+
+The Bareos project is fully committed to Open-Source.
+
+There are different types of Bareos binaries:
+
+#. Bareos Community binaries on https://download.bareos.org/:
+
+   * Publically accessable
+   * Latest build of the most recent Bareos stable branch at https://download.bareos.org/current/
+
+      * Only the latest build is available.
+      * Packages may be marked as prereleases (``<next-version-number>~pre``). However, packages are only published when they succeeded an automated testing process.
+      * When a new Bareos major version gets released, the version in this repository will also change to the new version.
+
+   * Latest build of the Bareos master branch at https://download.bareos.org/next/
+
+      * Only the latest build is available.
+
+#. Bareos Subscription binaries on https://download.bareos.com/
+
+   * Contain the repositories for the Bareos Subscription customers.
+   * The last three major releases are maintained and available in the corresponding subdirectory under https://download.bareos.com/bareos/release/
+   * Older versions stay available.
+   * While the repository can be browsed, using them do require authentication credentials, which come with a Bareos subscription.
+
+The software in both types of repositories is based on the same source code in https://github.com/bareos/bareos/. There are no hidden or Open-Core components.
+
+For a simple comparison of the two editions, please see the following table:
+
++-----------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+|          	        |  Community Edition                                                           |    Bareos Subscription                                                         |
++=======================+==============================================================================+================================================================================+
+|  **Packages**                                                                                                                                                                         |
++-----------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+| Last major releases   | (Source code only)                                                           | `Subscription repository <https://download.bareos.com/bareos/release/>`_       |
```

Comment:
```suggestion
| Previous major releases   | (Source code only)                                                           | `Subscription repository <https://download.bareos.com/bareos/release/>`_       |
```
Maybe using Previous disambigue what "last" can cover, because last can include current.